src/BenchmarkDotNet/Templates/MonoAOTLLVMCsProj.txt

```diff
@@ -0,0 +1,70 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" DefaultTarget="Publish">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>bin</OutputPath>
+    <TargetFramework>$TFM$</TargetFramework>
+    <MicrosoftNetCoreAppRuntimePackDir>$RUNTIMEPACK$</MicrosoftNetCoreAppRuntimePackDir>
+    <RuntimeIdentifier>$RUNTIMEIDENTIFIER$</RuntimeIdentifier>
+    <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
+    <PublishTrimmed>false</PublishTrimmed>
+    <AssemblyName>$PROGRAMNAME$</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Runtime.MonoAOTCompiler.Task" version="6.0.0-preview.3.21151.8" GeneratePathProperty="true" />
```

Comment:
We should most probably try to avoid using a hardcoded version? Should this be configurable? Or using the latest by having the asterisk?

```suggestion
    <PackageReference Include="Microsoft.NET.Runtime.MonoAOTCompiler.Task" version="6.0.0-*" GeneratePathProperty="true" />
```